media-libs/opencv/patches/opencv-3.4.3-source2.patchset

```diff
@@ -0,0 +1,24 @@
+From b90988d972f582acd6bb0b1782198ea7b2204519 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sat, 3 Nov 2018 21:11:38 +0100
+Subject: Build fix
+
+
+diff --git a/opencv_contrib-3.4.3/modules/sfm/src/libmv_light/libmv/numeric/numeric.cc b/opencv_contrib-3.4.3/modules/sfm/src/libmv_light/libmv/numeric/numeric.cc
+index 9007663..cd9bd86 100644
+--- a/opencv_contrib-3.4.3/modules/sfm/src/libmv_light/libmv/numeric/numeric.cc
++++ b/opencv_contrib-3.4.3/modules/sfm/src/libmv_light/libmv/numeric/numeric.cc
+@@ -21,6 +21,10 @@
+ 
+ #include "libmv/numeric/numeric.h"
+ 
++#ifdef __HAIKU__
++void		sincos(double, double *, double *);
```

Comment:
Do we even have an implementation for this function? Adding a prototype here may fix the build, but it'll render the module useless due of the missing symbol.

Reply:
I think we dont have implementatin, you are right. Let me research it deeper.

Reply:
I planned to tranplant it from openbsd, but i had to see that libroot already have sincos symbol, but no header file mentions it. What should we do now?

Reply:
Just add the declaration to headers/gnu/math.h, under a _GNU_SOURCE guard.